internal/tsoptions/tsconfigparsing.go

```diff
@@ -376,18 +367,19 @@ func convertJsonOption(
 		}
 	}
 	if isCompilerOptionsValue(opt, value) {
-		optType := opt.Kind
-		if optType == "list" {
+		switch opt.Kind {
+		case CommandLineOptionTypeList:
 			return convertJsonOptionOfListType(opt, value, basePath, propertyAssignment, valueExpression, sourceFile) // as ArrayLiteralExpression | undefined
-		} else if optType == "listOrElement" {
+		case CommandLineOptionTypeListOrElement:
 			if reflect.TypeOf(value).Kind() == reflect.Slice {
 				return convertJsonOptionOfListType(opt, value, basePath, propertyAssignment, valueExpression, sourceFile)
 			} else {
 				return convertJsonOption(opt.Elements(), value, basePath, propertyAssignment, valueExpression, sourceFile)
 			}
-		} else if !(reflect.TypeOf(optType).Kind() == reflect.String) {
```

Comment:
`Strada` uses a Map as type of enum option and use `!isString(opt.type)` as condition of this branch. However `tsgo` use `CommandLineOptionKind` for `CommandLineOption.Kind` whose underlying type is string.

Reply:
I think this is the bug you referred to in another comment. 🤔 @jakebailey